### ISSUE TITLE
Cannot installing tap via makedeb (fail for apt cannot exec #!/usr/bin/bash on running tap.postinst)

### ISSUE THREAD

**Issue by mobin-2008** [hwittenborn/tap#53]
### Issue

full log:
````
mobin@Mobin-Mahdi-PC:~/MPR/tap$ makedeb -si
[#] Making package: tap 2.0.4-1 (‫پنجشنبه ۰۲ ژوئن ۲۲، ۱۶:۲۶:۴۱‬)
[#] Checking for missing dependencies...
[#] Retrieving sources...
  [->] Cloning tap git repo...
Cloning into '/home/mobin/MPR/tap/tap'...
remote: Enumerating objects: 672, done.
remote: Counting objects: 100% (672/672), done.
remote: Compressing objects: 100% (365/365), done.
remote: Total 672 (delta 349), reused 557 (delta 275), pack-reused 0
Receiving objects: 100% (672/672), 137.86 KiB | 203.00 KiB/s, done.
Resolving deltas: 100% (349/349), done.
Note: switching to '1bf644cb83bcb9ba607f9a35c90248acb390cca0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

[#] Validating source files with sha256sums...
    tap ... Skipped
[#] Extracting sources...
  [->] Creating working copy of tap git repo...
Cloning into 'tap'...
remote: Enumerating objects: 661, done.
remote: Counting objects: 100% (661/661), done.
remote: Compressing objects: 100% (288/288), done.
remote: Total 661 (delta 344), reused 658 (delta 343), pack-reused 0
Receiving objects: 100% (661/661), 136.30 KiB | 45.43 MiB/s, done.
Resolving deltas: 100% (344/344), done.
Note: switching to '1bf644cb83bcb9ba607f9a35c90248acb390cca0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

[#] Starting prepare()...
sed -i 's|{pkgver}|2.0.4|' ./tap.py
sed -i 's|{pkgver}|2.0.4|' ./man/tap.8.adoc
[#] Entering fakeroot environment...
[#] Starting package()...
install -Dm 755 ./tap.py '/home/mobin/MPR/tap/pkg/tap/usr/bin/tap'
./setup.py install --prefix='./usr' --root='/home/mobin/MPR/tap/pkg/tap' --install-layout='deb'
running install
/usr/lib/python3/dist-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
running build
running build_py
creating build
creating build/lib
creating build/lib/tap
copying tap/__init__.py -> build/lib/tap
copying tap/apt_fetch_packages.py -> build/lib/tap
copying tap/apt_install_packages.py -> build/lib/tap
copying tap/arg_check.py -> build/lib/tap
copying tap/autoremove.py -> build/lib/tap
copying tap/build_dependency_tree.py -> build/lib/tap
copying tap/builddir_del_error.py -> build/lib/tap
copying tap/cfg.py -> build/lib/tap
copying tap/check_version.py -> build/lib/tap
copying tap/clone_packages.py -> build/lib/tap
copying tap/colors.py -> build/lib/tap
copying tap/create_mpr_json_dict.py -> build/lib/tap
copying tap/exceptions.py -> build/lib/tap
copying tap/find_apt_package_version.py -> build/lib/tap
copying tap/generate_apt_styled_text.py -> build/lib/tap
copying tap/get_editor_name.py -> build/lib/tap
copying tap/help_menu.py -> build/lib/tap
copying tap/install.py -> build/lib/tap
copying tap/list.py -> build/lib/tap
copying tap/main.py -> build/lib/tap
copying tap/message.py -> build/lib/tap
copying tap/parse_control.py -> build/lib/tap
copying tap/parse_srcinfo.py -> build/lib/tap
copying tap/read_config.py -> build/lib/tap
copying tap/read_dpkg_status_file.py -> build/lib/tap
copying tap/read_mpr_cache.py -> build/lib/tap
copying tap/remove.py -> build/lib/tap
copying tap/review_build_files.py -> build/lib/tap
copying tap/root_check.py -> build/lib/tap
copying tap/run_loading_function.py -> build/lib/tap
copying tap/run_transaction.py -> build/lib/tap
copying tap/search.py -> build/lib/tap
copying tap/set_mpr_dependencies.py -> build/lib/tap
copying tap/split_dependency_string.py -> build/lib/tap
copying tap/update.py -> build/lib/tap
copying tap/upgrade.py -> build/lib/tap
copying tap/utils.py -> build/lib/tap
copying tap/version.py -> build/lib/tap
running install_lib
creating /home/mobin/MPR/tap/pkg/tap/usr/lib
creating /home/mobin/MPR/tap/pkg/tap/usr/lib/python3
creating /home/mobin/MPR/tap/pkg/tap/usr/lib/python3/dist-packages
creating /home/mobin/MPR/tap/pkg/tap/usr/lib/python3/dist-packages/tap
copying build/lib/tap/__init__.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/apt_fetch_packages.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/apt_install_packages.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/arg_check.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/autoremove.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/build_dependency_tree.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/builddir_del_error.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/cfg.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/check_version.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/clone_packages.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/colors.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/create_mpr_json_dict.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/exceptions.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/find_apt_package_version.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/generate_apt_styled_text.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/get_editor_name.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/help_menu.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/install.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/list.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/main.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/message.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/parse_control.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/parse_srcinfo.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/read_config.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/read_dpkg_status_file.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/read_mpr_cache.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/remove.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/review_build_files.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/root_check.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/run_loading_function.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/run_transaction.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/search.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/set_mpr_dependencies.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/split_dependency_string.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/update.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/upgrade.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/utils.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
copying build/lib/tap/version.py -> /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/__init__.py to __init__.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/apt_fetch_packages.py to apt_fetch_packages.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/apt_install_packages.py to apt_install_packages.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/arg_check.py to arg_check.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/autoremove.py to autoremove.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/build_dependency_tree.py to build_dependency_tree.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/builddir_del_error.py to builddir_del_error.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/cfg.py to cfg.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/check_version.py to check_version.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/clone_packages.py to clone_packages.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/colors.py to colors.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/create_mpr_json_dict.py to create_mpr_json_dict.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/exceptions.py to exceptions.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/find_apt_package_version.py to find_apt_package_version.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/generate_apt_styled_text.py to generate_apt_styled_text.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/get_editor_name.py to get_editor_name.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/help_menu.py to help_menu.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/install.py to install.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/list.py to list.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/main.py to main.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/message.py to message.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/parse_control.py to parse_control.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/parse_srcinfo.py to parse_srcinfo.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/read_config.py to read_config.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/read_dpkg_status_file.py to read_dpkg_status_file.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/read_mpr_cache.py to read_mpr_cache.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/remove.py to remove.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/review_build_files.py to review_build_files.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/root_check.py to root_check.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/run_loading_function.py to run_loading_function.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/run_transaction.py to run_transaction.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/search.py to search.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/set_mpr_dependencies.py to set_mpr_dependencies.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/split_dependency_string.py to split_dependency_string.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/update.py to update.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/upgrade.py to upgrade.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/utils.py to utils.cpython-310.pyc
byte-compiling /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap/version.py to version.cpython-310.pyc
running install_egg_info
running egg_info
creating tap.egg-info
writing tap.egg-info/PKG-INFO
writing dependency_links to tap.egg-info/dependency_links.txt
writing top-level names to tap.egg-info/top_level.txt
writing manifest file 'tap.egg-info/SOURCES.txt'
reading manifest file 'tap.egg-info/SOURCES.txt'
adding license file 'LICENSE'
writing manifest file 'tap.egg-info/SOURCES.txt'
Copying tap.egg-info to /home/mobin/MPR/tap/pkg/tap/./usr/lib/python3/dist-packages/tap-1.egg-info
Skipping SOURCES.txt
running install_scripts
SOURCE_DATE_EPOCH='1644777721' asciidoctor -b manpage ./man/tap.8.adoc -o '/home/mobin/MPR/tap/pkg/tap/usr/share/man8/tap.8'
install -Dm 555 'completions/tap.bash' '/home/mobin/MPR/tap/pkg/tap/usr/share/bash-completion/completions/tap'
install -Dm 644 'tap.cfg' '/home/mobin/MPR/tap/pkg/tap/etc/tap.cfg'
[#] Tidying install...
  [->] Removing libtool files...
  [->] Purging unwanted files...
  [->] Removing static library files...
  [->] Stripping unneeded symbols from binaries and libraries...
  [->] Compressing man and info pages...
[#] Checking for packaging issues...
[#] Creating package "tap"...
  [->] Setting up package metadata...
  [->] Generating control file...
  [->] Adding postinst file to package...
  [->] Compressing package...
[#] Leaving fakeroot environment.
[#] Finished making: tap 2.0.4-1 (‫پنجشنبه ۰۲ ژوئن ۲۲، ۱۶:۲۶:۴۵‬)
[#] Installing package tap...
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'tap' instead of '/home/mobin/MPR/tap/tap_2.0.4-1_amd64.deb'
The following packages were automatically installed and are no longer required:
  asciidoctor autoconf-archive autoconf-doc bison-doc flex-doc gnu-standards libfl2 libjim0.79 libruby3.0 libsigsegv2 m4-doc rake ruby ruby-asciidoctor
  ruby-net-telnet ruby-rubygems ruby-webrick ruby-xmlrpc ruby3.0 rubygems-integration
Use 'sudo apt autoremove' to remove them.
The following NEW packages will be installed:
  tap
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/45.5 kB of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 /home/mobin/MPR/tap/tap_2.0.4-1_amd64.deb tap amd64 2.0.4-1 [45.5 kB]
Selecting previously unselected package tap.
(Reading database ... 345616 files and directories currently installed.)
Preparing to unpack .../MPR/tap/tap_2.0.4-1_amd64.deb ...
Unpacking tap (2.0.4-1) ...
Setting up tap (2.0.4-1) ...
dpkg (subprocess): unable to execute installed tap package post-installation script (/var/lib/dpkg/info/tap.postinst): No such file or directory
dpkg: error processing package tap (--configure):
 installed tap package post-installation script subprocess returned error exit status 2
Errors were encountered while processing:
 tap
E: Sub-process /usr/bin/dpkg returned an error code (1)
[!] Failed to install built package(s).
````

OS: Devuan ceres (Debian sid without systemd)

### Tap version

Cannot install tap!

**Comment by mobin-2008** [comment]
Please using `#!/bin/bash` instand of `#!/usr/bin/bash`

**Comment by hwittenborn** [comment]
From the stuff I've read, `/usr/bin/env bash` should just be used. I haven't had time to work on Tap recently, but feel free to submit a PR if you'd like it fixed though.

**Comment by mobin-2008** [comment]
Fixed #54
